aispace2/jupyter/stepdomwidget.py

```diff
@@ -53,7 +55,7 @@ def __init__(self):
         super().__init__()
         self.on_msg(self.handle_custom_msgs)
 
-        self.sleep_time = 0.2
+        # self.sleep_time = 0.2
```

Comment:
You cannot delete this. Here is where we set the default sleep_time.
If you delete this, and when the user run this (image below), the visualization will hang up.
You cannot delete this. Here is where we set the default sleep_time.
If you delete this, and when the user run without setting sleep_time (image below), the visualization will hang up.


<img width="741" alt="image" src="https://user-images.githubusercontent.com/39361141/64058296-ac9edc00-cb5d-11e9-85fd-aa7ad53a78ec.png">